### ISSUE TITLE
Color selection dialog broken

### ISSUE THREAD

**Issue by astares** [guillep/pharo-core#6587]
Open the settings dialog and try to change a color (like text selection color or other)

One gets the red square:

![Screenshot from 2020-06-13 17-05-46](https://user-images.githubusercontent.com/5980033/84572131-4e638280-ad98-11ea-8ac3-4a4de4fade8c.png)

**Comment by astares** [comment]
Pharo9.0.0
Build information: Pharo-9.0.0+build.373.sha.bcea011a62438ddb6d2c07035c477a7e48ebcf3a (64 Bit)
Unnamed

**Comment by kasperosterbye** [comment]
The error was still present in Pharo9.0.0 build: 386, commit: 0b06d20 (Wed 2020-06-17).

It seems like it is the primitive
```smalltalk
primRenderScanline: edge with: fill
	"Start/Proceed rendering the current scan line"
	<primitive: 'primitiveRenderScanline' module: 'B2DPlugin'>
	^self primitiveFailed
```
in class `BalloonEngine` which fails. I get this by inserting a halt at:
```smalltalk
	] on: Error do: [:err |
		self setProperty: #errorOnDraw toValue: true.
		self setProperty: #drawError toValue: err freeze.
		self halt.
		^ self drawErrorOn: aCanvas
	]
```
in `Morph>>fullDrawOn:`.

I am not able to go further with this one.

**Comment by astares** [comment]
Looks like this is something @tesonep or @estebanlm have to check on vm / plugin building level

**Comment by estebanlm** [comment]
well, there is no more Balloon in Pharo9, maybe that's related :)

**Comment by kasperosterbye** [comment]
Yup, that sounds like a plausibel explanation. What is supposed to take its place?

**Comment by Ducasse** [comment]
Pavel do you have an idea?

**Comment by jecisc** [comment]
The color picker is working currently. Please reopen if you face this problem in the latest Pharo
